.github/workflows/release.yml

```diff
@@ -14,13 +13,44 @@ env:
   GOPRIVATE: github.com/getlantern
   S3_BUCKET: lantern
 jobs:
-    set-version:
-      runs-on: ubuntu-latest
-      outputs:
-        version: ${{ steps.set-version.outputs.version }}
-        prefix: ${{ steps.set-version.outputs.prefix }}
-        version_file: ${{ steps.set-version.outputs.version_file }}
-      steps:
+  determine-platform:
+    runs-on: ubuntu-latest
+    outputs:
+      platform: ${{ steps.set-platform.outputs.platform }}
+    steps:
+      - name: Determine Platform
+        id: set-platform
+        run: |
+          echo "GITHUB_REF is: $GITHUB_REF"
+          if [[ "$GITHUB_REF" == refs/tags/* ]]; then
+           TAG=${GITHUB_REF#refs/tags/}
+           echo "Tag is: $TAG"
+           if [[ $TAG == ios-* ]]; then
+             echo "Platform determined: ios"
+             echo "platform=ios" >> "$GITHUB_OUTPUT"
+           elif [[ $TAG == android-* ]]; then
+             echo "Platform determined: android"
+             echo "platform=android" >> "$GITHUB_OUTPUT"
+           elif [[ $TAG == desktop-* ]]; then
+             echo "Platform determined: macos"
+             echo "platform=desktop" >> "$GITHUB_OUTPUT"
```

Comment:
So with these changes, we can release the app on different platforms by varying what the tag is prefixed with? Cool! I would just add a note about that to the [Building release packages](https://github.com/getlantern/lantern-client?tab=readme-ov-file#building-release-packages) section of the README.

Reply:
Yeah, I will update the readme shortly.

Reply:
Here new readme [section](https://github.com/getlantern/lantern-client/tree/jigar/ios-ci?tab=readme-ov-file)